meshtastic/mesh.proto

```diff
@@ -1266,6 +1266,26 @@ message MeshPacket {
     DELAYED_DIRECT = 2;
   }
 
+  /*
+   * Enum to identify which transport mechanism this packet arrived over
+   */
+  enum TransportMechanism {
+    /*
+     * The default case is LoRa itself
+     */
+    TRANSPORT_LORA = 0;
```

Comment:
In a future where we have a 2.4GHz LoRA radio and a Sub GHz radio, did we want to separate those?